src/xFactor/base/Application.java

```diff
@@ -0,0 +1,245 @@
+		allParticipants.add(new Participant("Martin", "Grigorov", 25, Gender.Male, "Sofia"));
+		allParticipants.add(new Participant("Nataliya", "Vasileva", 25, Gender.Female, "Burgas"));
+		allParticipants.add(new Participant("Nayden", "Nikolov", 25, Gender.Male, "Varna"));
+		allParticipants.add(new Participant("Yasen", "Gurvanov", 25, Gender.Male, "Blagoevgrad"));
+		allParticipants.add(new Participant("Grudi", "Radev", 25, Gender.Male, "Stara Zagora"));
+		allParticipants.add(new Participant("Dilyana", "Stolarova", 25, Gender.Female, "Varna"));
+		allParticipants.add(new Participant("Toni", "Ilieva", 25, Gender.Female, "Troyan"));
+		allParticipants.add(new Participant("Zdravko", "Kostadinov", 25, Gender.Male, "Sofia"));
+		allParticipants.add(new Participant("Kristiyan", "Srebrev", 25, Gender.Male, "Purvomay"));
+	}
+
+	private void fillQualities() {
+		qualities = new ArrayList<String>();
+		qualities.add("Dancing");
+		qualities.add("Painting");
+		qualities.add("Playing the guitar");
+		qualities.add("Playing the piano");
+		qualities.add("Playing the viola");
+		qualities.add("Playing the drums");
+	}
+
+	private void initializeStage(int stageNum, int max, Stage stage) {
+		if (stageNum < 1 || stageNum > numberOfStages) {
+			throw new IllegalArgumentException("The number of stage must be between 1 and " + numberOfStages);
+		}
+
+		if (allStages.size() - 1 == numberOfStages) {
+			throw new IllegalArgumentException("Maximum number of stages reached! Cannot create a new one.");
+		} else if (allStages.size() == 1) {
+			stage.setMaxParticipants(allParticipants.size());
+			stage.getParticipantsInStage().addAll(participants);
+		} else {
+			stage.setMaxParticipants(max);
+		}
+		
+		stageDao.save(stage);
+	}
+
+	public void createJudges(int number) {
+		judges = new ArrayList<Judge>();
+		addJudges();
+		for (int i = 0; i < number; i++) {
+			judges.add(allJudges.get(i));
+		}
+
+		for (int i = 0; i < judges.size(); i++) {
+			judgeDao.save(judges.get(i));
+		}
+	}
+
+	public ArrayList<Judge> getJudges() {
+		return judges;
+	}
+
+	public void giveSpecialVote(int id) {
+		for (Judge judge : judges) {
+			if (judge.getJudgeId() == id) {
+				judge.setSpecial(true);
+				judgeDao.save(judge);
+			}
+		}
+	}
+
+	public Judge getSpecialJudge() {
+		for (Judge judge : judges) {
+			if (judge.isSpecial()) {
+				return judge;
+			}
+		}
+		return null;
+	}
+
+	public void createParticipants(int number) {
+		participants = new ArrayList<Participant>();
+		addParticipants();
+		for (int i = 0; i < number; i++) {
+			participants.add(allParticipants.get(i));
+		}
+
+		for (int i = 0; i < participants.size(); i++) {
+			participantDao.save(participants.get(i));
+		}
+	}
+
+	public Stage createStage(int stageNum, int max) {
+		Stage stage = new Stage(stageNum, max);
+		allStages.add(stage);
+		initializeStage(stageNum, max, stage);
+		stageDao.save(stage);
+		return stage;
+	}
+
+	public Set<Participant> getParticipantsInStage(Stage stage) {

```

Comment:
this should be either get or save, it is not good practice for one method to do 2 things at the same time